ui/src/masking.ts

```diff
@@ -77,7 +77,7 @@ export function maskUserInfo(unkonwnArg: unknown, actionValuesToMask: string[],
     }
 
     data = data.replace(/[a-z]+:\/\/\S*/gi, getRedactedLabel('url'));
-    data = data.replace(/\S*\.(com|org|net)\S*/gi, getRedactedLabel('url'));
+    data = data.replace(/\S+(?<!(?<!\-)\basp)\.(com|org|net)\S*/gi, getRedactedLabel('url'));
```

Comment:
Please add ".NET" and "ASP.NET" to the unit tests - https://github.com/microsoft/vscode-azuretools/blob/main/ui/test/masking.test.ts

Reply:
I added tests for ".NET", "ASP.NET", "microsoft-asp.net", "microsoftasp.net", and a few flavors of those.